forc-plugins/forc-doc/src/render.rs

```diff
@@ -185,43 +201,78 @@ fn html_body(
     code_str: String,
     item_attrs: String,
 ) -> Box<dyn RenderBox> {
-    let href = if module_depth > 0 {
-        "../all.html"
-    } else {
-        "../doc/all.html"
-    }
-    .to_string();
+    let mut all_path = module_depth_to_path_prefix(module_depth);
+    all_path.push_str("all.html");
```

Comment:
It looks like we have an `INDEX_FILENAME` const for this literal, but it's currently scoped within forc-doc's main function. Let's move it somewhere so that it can be shared between `main.rs` and here.